ui/app/templates/components/dashboard/secrets-engines-card.hbs

```diff
@@ -0,0 +1,69 @@
+<div class="is-flex-between has-left-margin-xs" data-test-dashboard-secrets-engines-header>
+  <h3 class="title is-4">Secrets Engines</h3>
+  <div>
+    <Hds::Link::Inline @route="vault.cluster.secrets.backends" class="has-text-weight-semibold is-no-underline">
+      View more
+    </Hds::Link::Inline>
+  </div>
+</div>
+<Hds::Table @caption="Five secrets engines" class="has-border-collapse" data-test-dashboard-secrets-engines-table>
```

Comment:
As mentioned, not everyone is going to have five secrets engines. They may just have one. Not sure what to replace this caption with, but something to talk with design about?

Also, maybe I'm missing something, but is this caption supposed to show?
![image](https://github.com/hashicorp/vault/assets/6618863/a1739007-ad06-45c7-9df7-f34208b977bd)

Reply:
@Monkeychip Which caption are you referencing? The Vault version will be tackled on a separate ticket, but the description for the secret engine is supposed to show :)

Reply:
Just left a comment on Figma for design!